ext/yarp/extension.c

```diff
@@ -436,6 +436,24 @@ compile(VALUE self, VALUE string) {
     return result;
 }
 
+static VALUE
+newlines(VALUE self, VALUE string) {
```

Comment:
Can we add a comment describing this method? Thinking specifically it'd be helpful to say that `string` is the actual text of the file, not the filename

Reply:
👍